lib/block-supports/layout.php

```diff
@@ -184,9 +184,9 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 				);
 			}
 		} else {
-			$layout_styles[ $selector ] = array(
-				'flex-direction' => 'column',
-				'align-items'    => 'flex-start',
+			$layout_styles[] = array(
+				'selector'     => $selector,
+				'declarations' => array( 'flex-direction' => 'column' ),
```

Comment:
Not sure, but should the align-items prop also come across?

```suggestion
				'declarations' => array( 'flex-direction' => 'column', 'align-items' => 'flex-start' ),
```

Reply:
It looks like we already have a duplicate `'align-items' => 'flex-start'` in the else condition below, so I think we were duplicating that output previously?